openapi.yaml

```diff
@@ -0,0 +1,139 @@
+openapi: 3.0.0
```

Comment:
Does this file work with our developer portal? I thought smartdocs doesn't support openapi 3 yet.